dashboard/test/ui/features/craft/hero.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Minecraft hero
+
+  Scenario: Signed out finish dialog
+    Given I am on "http://studio.code.org/s/hero/stage/1/puzzle/12?noautoplay=true"
+    And I wait for the page to fully load
+    And I wait until the Minecraft game is loaded
+    And I press "runButton"
+    And I click selector "button:contains(Finish)" once I see it
+    And I wait until element "#continue-button" is visible
+    And element "#publish-to-project-gallery-button" is not visible
+    And element "#save-to-project-gallery-button" is not visible
+
+    # capture finish dialog in saucelabs video
+    And I wait for 1 second
```

Comment:
Is there an element we can wait for instead? Time based waits tend to me more finicky.

Reply:
not really -- all the elements have already appeared in the DOM, so we are literally waiting for time to pass so that whatever is on the screen shows up in the video before the test ends. I believe the comment captures this